Source/Fuse.Navigation/WhileCanGoBack.uno

```diff
@@ -11,6 +11,8 @@ namespace Fuse.Navigation
 	public abstract class WhileHistoryTrigger : WhileTrigger
 	{
 		IBaseNavigation _context;
+		//TODO: This won't work due to being overidden in `OnUnrooted`. A good fix would be to remove this `public` part
+		//and alter the `Navigation` override to work with a `Node` (if that's even possible)
```

Comment:
I meant the other way around, so we'd keep the comment if we needed to revert the other part. But sure, it's not a big deal.